pkg/appfile/parser.go

```diff
@@ -36,6 +40,9 @@ type Workload struct {
 	Template           string
 	HealthCheckPolicy  string
 	CustomStatusFormat string
+
+	Helm         *v1alpha2.Helm
+	DefReference v1alpha2.DefinitionReference
```

Comment:
```suggestion
	DefinitionReference v1alpha2.DefinitionReference
```

The code will be hard to read when we use many abbreviations that has not come into consensus.